packages/babel-helper-fixtures/src/index.js

```diff
@@ -141,11 +141,14 @@ export default function get(entryLoc): Array<Suite> {
       const taskOptsLoc = resolve(taskDir + "/options");
       if (taskOptsLoc) extend(taskOpts, require(taskOptsLoc));
 
+      const taskLogLoc = resolve(taskDir + "/expected-log.txt");
```

Comment:
The `debug-fixtures` in preset-env is testing against `stdout.txt` and `stderr.txt`. I think we could keep consistency here? And it would be great if preset-env does not log to test output after this PR gets merged.

Reply:
> The `debug-fixtures` in preset-env is testing against `stdout.txt` and `stderr.txt`. I think we could keep consistency here?

I like that, by using the same file, you are also keeping the messages ordered. I would slightly prefer to replace `stdout.txt` and `stderr.txt` with `expected-log.txt`, but I don't have a string opinion.

> And it would be great if preset-env does not log to test output after this PR gets merged.

:+1: